builtin_interfaces/Time stamp
uint16[] id_list

bool[] torque_enable
string[] drive_mode
string[] operating_mode